src/ethereum_spec_tools/lint/__init__.py

```diff
@@ -11,11 +11,45 @@
 import pkgutil
 from abc import ABCMeta, abstractmethod
 from dataclasses import dataclass
+from itertools import zip_longest
 from typing import Generator, List, Optional, Sequence, Tuple
 
 from ..forks import Hardfork
 
 
+def compare_ast(old: ast.AST, new: ast.AST) -> bool:
```

Comment:
If we want to put `compare_ast` in the superclass (and not in a specific lint), we want it to be generic. Maybe something like this, where `ignore` is called whenever two nodes differ and returns `True` if that difference should be ignored:

```python
def compare_ast(old: ast.AST, new: ast.AST, ignore: Callable[[ast.AST, ast.AST], bool]):
    pass
```

That said, I think I'd prefer the quick and dirty option of putting `compare_ast` in `glacier_forks_hygiene.py` file for now. It's simpler and we don't need to worry about making it generic until we actually need another lint that compares syntax trees.

Reply:
ok.